pkgs/development/python-modules/blspy/default.nix

```diff
@@ -40,6 +41,12 @@ buildPythonPackage rec {
         fetchSubmodules = true;
       };
     })
+
+    # https://github.com/Chia-Network/bls-signatures/issues/286
```

Comment:
do you mind adding a comment about what this does?

Reply:
sure!